azext_edge/edge/providers/support/orc.py

```diff
@@ -16,62 +16,54 @@
     process_deployments,
     process_v1_pods,
     process_services,
-    process_replicasets
+    process_replicasets,
 )
 
 logger = get_logger(__name__)
 
 
-ORC_INSTANCE_LABEL = "app.kubernetes.io/instance in (azure-iot-operations)"
-ORC_APP_LABEL = "app in (aio-orc-api)"
+ORC_APP_LABEL = "app in (aio-orc-api, cert-manager, cainjector, webhook)"
 ORC_CONTROLLER_LABEL = "control-plane in (aio-orc-controller-manager)"
 
 # TODO: @jiacju - this label will be used near future for consistency
```

Comment:
Hi @digimaun, if no orc consistent label will be added, should we remove this TODO?🤔

Reply:
I thought we were going to be hopeful about the service team doing the right thing?

Reply:
but sounds like they weren't on the agreement in the first place?

Reply:
Right orchestrator wasn't part of common labeling